tests/test_view/test_model_views.py

```diff
@@ -0,0 +1,207 @@
+        for test in self.tests["get_all"]:
+            self.data_update(test)
+            resp = self.send_request(f"/{self.name}", client, token=token)
+            self.process_resp(resp, test, True)
+
+    def test_get_count(self, client: TestClient, token: str):
+        for test in self.tests["get_count"]:
+            self.data_update(test)
+            resp = self.send_request(f"/{self.name}/count", client, token=token)
+            self.process_resp(resp, test)
+
+    def test_get_one(self, client: TestClient, token: str):
+        for test in self.tests["get_one"]:
+            self.data_update(test)
+            resp = self.send_request(f"/{self.name}/{test['obj_id']}", client, token=token)
+            self.process_resp(resp, test)
+
+    def test_partial_update(self, client: TestClient, token: str):
+        for test in self.tests["partial_update"]:
+            self.data_update(test)
+            resp = self.send_request(
+                f"/{self.name}/{test['obj_id']}",
+                client,
+                json=test["data"],
+                method="patch",
+                token=token,
+            )
+            self.process_resp(resp, test)
+
+    def test_full_update(self, client: TestClient, token: str):
+        for test in self.tests["full_update"]:
+            self.data_update(test)
+            resp = self.send_request(
+                f"/{self.name}/{test['obj_id']}",
+                client,
+                json=test["data"],
+                method="put",
+                token=token,
+            )
+            self.process_resp(resp, test)
+
+    def test_delete(self, client: TestClient, token: str):
+        for test in self.tests["delete"]:
+            self.data_update(test)
+            resp = self.send_request(f"/{self.name}/{test['obj_id']}", client, method="delete", token=token)
+            self.process_resp(resp, test)
+
+
+class TestUsers(ViewTestMixin):
+    name = "users"
+    auth = True
+    tests = json_module.loads(open("tests/fixtures/data/users.json").read())
+
+
+class TestDiscounts(ViewTestMixin):
+    name = "discounts"
+    auth = True
+    tests = json_module.loads(open("tests/fixtures/data/discounts.json").read())
+
+
+class TestNotifications(ViewTestMixin):
+    name = "notifications"
+    auth = True
+    tests = json_module.loads(open("tests/fixtures/data/notifications.json").read())
+
+
+class TestTemplates(ViewTestMixin):
+    name = "templates"
+    auth = True
+    tests = json_module.loads(open("tests/fixtures/data/templates.json").read())
+
+
+class TestWallets(ViewTestMixin):
+    name = "wallets"
+    auth = True
+    tests = json_module.loads(open("tests/fixtures/data/wallets.json").read())
+
+
+class TestStores(ViewTestMixin):
+    name = "stores"
+    auth = True
+    tests = json_module.loads(open("tests/fixtures/data/stores.json").read())
+
+    @pytest.fixture(scope="class", autouse=True)
+    def setup(self, wallet, notification):
```

Comment:
What is this for?
Also, is `data_update` hook reserved for the future? What is it intended to do? Load related?